tests/unit/amo/components/TestCategory.js

```diff
@@ -500,6 +500,34 @@ describe(__filename, () => {
     );
   });
 
+  it('renders a theme class name if type is a theme', () => {
+    const root = render(
+      { params: { visibleAddonType: visibleAddonType(ADDON_TYPE_THEME) } },
+      {
+        autoDispatchCategories: false,
+        paramOverrides: {
+          visibleAddonType: visibleAddonType(ADDON_TYPE_THEME),
+        },
+      },
+    );
+
+    expect(root).toHaveClassName('Category--theme');
+  });
+
+  it('renders without a theme class name if type is an extension', () => {
+    const root = render(
+      { params: { visibleAddonType: visibleAddonType(ADDON_TYPE_EXTENSION) } },
```

Comment:
Do you need to specify both `params` and `paramOverrides`, that seems odd.

Reply:
oh! will double check : /